tests/pytorch/fused_attn/test_fused_attn_with_cp.py

```diff
@@ -123,6 +123,14 @@ def test_cp_with_fused_attention(dtype, model, qkv_format, cp_comm_type):
         pytest.skip(
             f"Fused attention does not support sliding window attention + context parallelism yet!"
         )
+    if cp_comm_type == "all_gather" and dtype == "fp8":
+        pytest.skip(
+            f"CP implementation with KV all-gather does not support FP8 + context parallelism yet!"
+        )
+    if dtype == "fp8" and qkv_format == "thd":
+        pytest.skip(f"FP8 attention cannot work with THD format yet!")
+    if dtype == "fp8" and config.attn_bias_type != "no_bias":
+        pytest.skip(f"FP8 attention cannot work with bias yet!")
```

Comment:
We probably don't need the `f` here but it's a small nitpick.

Reply:
Fixed.